docs/user/README.md

```diff
@@ -57,7 +57,7 @@ $ cd rust-analyzer
 $ cargo install --path ./crates/ra_lsp_server/ --force --locked
 $ cd ./editors/code
 $ npm install
-$ ./node_modules/vsce/out/vsce package
+$ npm run-script package
```

Comment:
This is much more common to use `npm run` nowadays ([this just aliases the same subcommand](https://docs.npmjs.com/cli/run-script))
```suggestion
$ npm run package
```